Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -9742,6 +9847,54 @@ def main():
                     empty_result_message="No template stacks found."
                 )
             )
+        elif demisto.command() == 'pan-os-platform-get-global-counters':
```

Comment:
There is a variable called command, so instead of calling demisto.command() every time use that variable instead.

Should be changed to the commands from previous PR as well.

Reply:
True, fixed